### ISSUE TITLE
CupertinoDateTime Picker not changing format when language is changed

### ISSUE THREAD

**Issue by parth22** [flutter/flutter#44412]
the date format in ios 13 look like
"Wed 6 Nov"
![Image from iOS](https://user-images.githubusercontent.com/13308845/68446615-3d9bce80-0203-11ea-9673-20271a19fab8.png)
but when we apply widgets it's different
"Wed Nov 6"
![Screen Shot 2019-11-07 at 12 39 31](https://user-images.githubusercontent.com/13308845/68446638-52786200-0203-11ea-9f04-d1097bb35d82.png)

My flutter doctor output:
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.9.1+hotfix.6, on Mac OS X 10.13.6 17G65, locale
    en-IN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.1)
[✓] Xcode - develop for iOS and macOS (Xcode 10.1)
[✓] Android Studio (version 3.5)
[✓] VS Code (version 1.39.2)
[✓] Connected device (1 available)

• No issues found!

**Comment by LongCatIsLooong** [comment]
After switching to en-IN, the gallery app doesn't seem to change date format while native apps do.

**Comment by parth22** [comment]
@LongCatIsLooong Yes, I've check some native apps of ios 13 the date time picker looks same as i mention "Wed 6 Nov" but, using flutter sdk format change to  "Wed Nov 6".
